src/main/java/org/folio/edge/orders/ValidateHandler.java

```diff
@@ -27,7 +28,8 @@ protected void handle(RoutingContext ctx) {
           logger.info("Request is from purchasing system: " + ps.toString());
           ((OrdersOkapiClient) client).validate(ctx.request().method(),
               ps,
-              ctx.request().headers(),
+              // the responses are short, we can safely drop the encoding header if passed
+              ctx.request().headers().remove(HttpHeaders.ACCEPT_ENCODING),
```

Comment:
I think we probably want to do this for all requests, not just /orders/validate...  Maybe you can handle this in a single place in the OkapiClient